apps/api/src/app/events/services/push-service/handlers/fcm.handler.ts

```diff
@@ -0,0 +1,16 @@
+import { ChannelTypeEnum } from '@novu/shared';
+import { FcmPushProvider } from '@novu/fcm';
+import { BasePushHandler } from './base.handler';
+import { ICredentials } from '@novu/dal';
+
+export class FCMHandler extends BasePushHandler {
+  constructor() {
+    super('fcm', ChannelTypeEnum.PUSH);
+  }
+
+  buildProvider(credentials: ICredentials) {
+    this.provider = new FcmPushProvider({
+      secretKey: credentials.secretKey,
```

Comment:
Is only a secretKey is needed here? No other info?

Reply:
Yes. The secret key in this case it the service account key JSON.

Reply:
Oh so that's why it was strange for me, since I've expected secretKey to be a property of the json rather than the whole object itself.